File: LIPA/Fontes_POO/C_SHARP/exemplo7.cs
using System;

public class Program
{
	
  public class CLS_CALCULO
  {
    
    public int SOMAR(int A, int B)
    {
      return A + B;
    }

    public double SOMAR(double A, double B)
    {
      return A + B;
    }
    
  }

  public static void Main(string[] args)
  {
    
    CLS_CALCULO OPERACAO = new CLS_CALCULO();

    Console.WriteLine(OPERACAO.SOMAR(1.3, 2.7));
    Console.WriteLine(OPERACAO.SOMAR(2, 2.5));
    Console.WriteLine(OPERACAO.SOMAR(1.5, 6));
    Console.WriteLine(OPERACAO.SOMAR(4, 2));
	
  }
  
}

File: LIPA/Fontes_POO/JAVA/exemplo7.java
public class Main {
	
  public static class CLS_CALCULO {
    
    public int SOMAR(int A, int B) {
      return A + B;
    }

    public double SOMAR(double A, double B) {
      return A + B;
    }
    
  }

  public static void main(String[] args) {
	  
    CLS_CALCULO OPERACAO = new CLS_CALCULO();

    System.out.println(OPERACAO.SOMAR(1.3, 2.7));
    System.out.println(OPERACAO.SOMAR(2, 2.5));
    System.out.println(OPERACAO.SOMAR(1.5, 6));
    System.out.println(OPERACAO.SOMAR(4, 2));
	
  }
  
}
